test/Tests/src/Data/Numbers_Spec.enso

```diff
@@ -220,6 +230,12 @@ spec =
             Decimal.parse "000000.0001" . should_equal 0.0001
             Decimal.parse "aaaa" . should_fail_with Number_Parse_Error.Error
 
+        Test.specify "decimal should parse hundred factorial well" <|
+            txt = hundred_factorial.to_text + ".345"
+            decimal = Decimal.parse txt
+            is_huge = decimal > (hundred_factorial / 5)
```

Comment:
```suggestion
            is_huge = decimal > hundred_factorial
```

Why the `/ 5`? Seems like it should be larger than the factorial itself? Or is it due to lack of precision in decimals?

Still, seems like `0.9 * hundred_factorial` should be more than enough, vs `0.2`, no?

Reply:
I am not sure what are the _deltas_ between `double` values at that size. Yes, probably `0.9` would work. But I just wanted to make sure something big is being computed.

Reply:
:ok: